ui/raidboss/data/06-ew/raid/p2s.ts

```diff
@@ -0,0 +1,311 @@
+      },
+    },
+    {
+      id: 'P2S Mark of the Tides',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AD0' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, matches, output) => output.awayFromGroup!(),
+      outputStrings: {
+        awayFromGroup: Outputs.awayFromGroup,
+      },
+    },
+    {
+      id: 'P2S Mark of the Depths',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AD1' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, matches, output) => output.stackOnYou!(),
+      outputStrings: {
+        stackOnYou: Outputs.stackOnYou,
+      },
+    },
+    {
+      id: 'P2S Channeling Flow',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['AD2', 'AD3', 'AD4', 'AD5'] }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, matches, output) => {
+        const t = parseFloat(matches.duration);
+        if (t < 15)
+          return output.shortTide!();
+        return output.longTide!();
+      },
+      outputStrings: {
+        shortTide: {
+          en: 'Short Tide',
+        },
+        longTide: {
+          en: 'Long Tide',
+        },
+      },
+    },
+    {
+      // Aoe from head outside the arena
+      id: 'P2S Dissociation',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '682C', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '682C', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '682C', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '682C', source: 'ヒッポカムポス' }),
+      alertText: (_data, matches, output) => {
+        const xCoord = parseFloat(matches.x);
+        if (xCoord < 100)
+          return output.w!();
+        if (xCoord > 100)
+          return output.e!();
+      },
+      outputStrings: {
+        e: Outputs.east,
+        w: Outputs.west,
+      },
+    },
+    {
+      // Spread aoe marker on some players, not all
+      id: 'P2S Tainted Flood',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6838', source: 'Hippokampos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6838', source: 'Hippokampos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6838', source: 'Hippokampos' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6838', source: 'ヒッポカムポス' }),
+      condition: (data, matches) => matches.target === data.me,
+      response: Responses.spread(),
+    },
+    {
+      id: 'P2S Coherence Flare',
+      type: 'HeadMarker',
+      // Whoever has tether when cast of 681B ends will be flared
+      netRegex: NetRegexes.headMarker({ id: '0054' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      run: (data, matches) => data.flareTarget = matches.target,
+      outputStrings: {
+        text: {
+          en: 'Tank flare tether on YOU',

```

Comment:
Mostly suggesting to keep this as short as possible because it could hit other people.

```suggestion
          en: 'Flare Tether',
```